packages/pyroscope-flamegraph/src/FlameGraph/FlameGraphComponent/Tooltip.tsx

```diff
@@ -11,6 +11,8 @@ import {
 
 import { DefaultPalette, FlamegraphPalette } from './colorPalette';
 import styles from './Tooltip.module.scss';
+import Icon from '@webapp/ui/Icon';
```

Comment:
Yeah this is a circular dependency, not entirely sure how it affects the bundle size.

If it affects negatively I would recommend to just inline this Icon.
But yeah in the near future we should make another package just for ui.

Reply:
I do think we should keep the icon. I will give a different one in slack that is an SVG like the one in the image